lib/saito/ui/saito-header/saito-header.js

```diff
@@ -111,9 +111,10 @@ class SaitoHeader extends UIModTemplate {
       //
       let add_other_cryptos_html = `<option id="crypto-option-add-new" value="add-new">install new crypto...</option>`;
       this.app.browser.addElementToElement(add_other_cryptos_html, document.querySelector("#header-token-select"));  
+      // console.log("renderCrypto2", document.querySelector("#header-token-select").innerHTML);
 
     } catch (err) {
-
+      console.log(err);
```

Comment:
Should this still be here?  
If yes, would this be better?: `console.error(err)`